pystiche_papers/ulyanov_et_al_2016/_loss.py

```diff
@@ -118,9 +135,14 @@ def __init__(
         # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/src/texture_loss.lua#L38
         # In the reference implementation the gram_matrix is only divided by the
         # batch_size.
-        normalize = not impl_params
         self.normalize_by_num_channels = impl_params
-        super().__init__(encoder, normalize=normalize, **gram_op_kwargs)
+        super().__init__(encoder, **gram_op_kwargs)
```

Comment:
`normalize=True` by default. I think you mean to set `normalize=False`, didn't you?

Reply:
Yeah, you're right, I removed that by mistake. They are not using a normalisation in their gram calculation.